core/controllers/question_editor_test.py

```diff
@@ -25,14 +25,14 @@
 (question_models,) = models.Registry.import_models([models.NAMES.question])
 
 
-class QuestionsHandlersTest(test_utils.GenericTestBase):
+class EditableQuestionDataHandlerTest(test_utils.GenericTestBase):
```

Comment:
Make sure all functions in question_editor is tested.

Reply:
added all tests